test/integration_test.go

```diff
@@ -126,6 +126,13 @@ func (s *IntegrationSuite) TestSimpleSharding() {
 		assert.True(t, affected <= 1)
 	}
 
+	t.Run("Count", func(t *testing.T) {
+		row := db.QueryRow("select count(*) as ttt from student")
+		var cnt int64
+		assert.NoError(t, row.Scan(&cnt))
+		assert.Equal(t, int64(100), cnt)
+	})
+
```

Comment:
Move it to a single function of  test suite.

Reply:
address